scripts/check_git_status.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+if [ $(git status -s | wc -c) -gt 0 ];
+then
+    echo "The git repository state changed after the build, this should not happen."
```

Comment:
Let's print the output of `git status` too, please.

Reply:
Very good idea